scripts/gcovr

```diff
@@ -515,6 +515,11 @@ def process_gcov_data(data_fname, covdata, source_fname, options):
             fname = aliases.unalias_path(os.path.join(common_dir, segments[-1].strip()))
         else:
             fname = aliases.unalias_path(os.path.join(common_dir, segments[-1]).strip())
+        if not os.path.exists(fname):
+            # let's try using the path to the gcov file as base directory
+            # Test before assigning not to change behavior compared to previous versions
+            if os.path.exists(os.path.join(os.path.dirname(data_fname), segments[-1].strip())):
+                fname = aliases.unalias_path(os.path.join(os.path.dirname(data_fname), segments[-1].strip()))
```

Comment:
Can we assign the path to some variable to avoid code duplication? E.g.

``` python
possible_gcov_path = os.path.join(...)
if os.path.exists(possible_gcov_path):
  fname = aliases.unalias_path(possible_gcov_path)
```

Reply:
Done